presidio-anonymizer/tests/test_anonymizer_result.py

```diff
@@ -0,0 +1,24 @@
+
+from presidio_anonymizer.entities import AnonymizerResult
+from presidio_anonymizer.entities.anonymizer_result_item import AnonymizerResultItem
+
```

Comment:
I would add tests for the equal, as_json and add_item to verify that if anyone changes them in the future and break something, the developer will be aware.

Reply:
I added more tests like you suggested except from as_json because it contains no logic at all.